docs/io/votable/dataorigin.rst

```diff
@@ -0,0 +1,101 @@
+.. doctest-remote-data-all::
+
+Introduction
+------------
+
+Extract basic provenance information from VOTable header. The information is described in
+DataOrigin IVOA note: https://www.ivoa.net/documents/DataOrigin/.
+
+DataOrigin includes both the query information (such as publisher, contact, versions, etc.)
+and the Dataset origin (such as Creator, bibliographic links, URL, etc.)
+
+This API retrieves Metadata from INFO in VOTable.
+
+
+Getting Started
+---------------
+
+To extract DataOrigin from VOTable
+
+Example: VizieR catalogue J/AJ/167/18
+
```

Comment:
to avoid sphinx crashing, see https://github.com/scientific-python/pytest-doctestplus/issues/291

```suggestion
Introduction
------------

Extract basic provenance information from VOTable header. The information is described in
DataOrigin IVOA note: https://www.ivoa.net/documents/DataOrigin/.

DataOrigin includes both the query information (such as publisher, contact, versions, etc.)
and the Dataset origin (such as Creator, bibliographic links, URL, etc.)

This API retrieves Metadata from INFO in VOTable.


Getting Started
---------------

To extract DataOrigin from VOTable

Example: VizieR catalogue J/AJ/167/18

.. doctest-remote-data-all::

```